### DISCUSSION TITLE
Refresh cache would be great

### DISCUSSION THREAD

**Discussion by Roland80** [rappen/FetchXMLBuilder#934] [Ideas]
As much as I hate having to wait to get started building a query, I love that data is cached. However, when an environment is being worked on, tables and fields can be added or changed as you're working. This means the cache can get outdated regularly and I have to close and restart XRM Toolbox to be able to access a new table or field, as closing and re-opening FetchXML Builder doesn't clear the cache.

Ir'd be awesome if we could clear the cache on a per-table basis, but I'd already be extremely happy if we could clear the entire cache per environment/connection, without having to close the entire toolbox.

**Comment by rappen** [comment]
I totally agree!
Related to #897.

I'll try to find a good solution for this. Stay tuned...
